istioctl/pkg/writer/ztunnel/configdump/testdata/workloadsummary.txt

```diff
@@ -0,0 +1,28 @@
+NAME                                                 NAMESPACE          IP NODE                  WAYPOINT                            PROTOCOL
```

Comment:
Isn't there a wront indent here for `IP NODE`?

Reply:
Just verified it's not, `IP` and `NODE` are separate columns. However I think workloads' IPs are not read correctly since the field has changed as well, I'll update with a new fix.

Reply:
Updated, can you check if https://github.com/istio/istio/pull/46384/files#diff-29c6d27acf31e7377e380900dd095fa95a57c461dc804749a9d190f8cad17a70 makes more sense? Thanks.

Reply:
Yes, thanks.